runtime/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ if (NOT TPP_INSIDE_IREE)
   target_compile_definitions(tpp_c_runner_utils PRIVATE mlir_c_runner_utils_EXPORTS)
 else()
   add_library(tpp_c_runner_utils
-    STATIC
+    SHARED
```

Comment:
Why shared here?

Reply:
Because for IREE plugin it needs to be a shared object that gets loaded at runtime.

We need to change the CMake files too (on a separate PR), to remove references to IREE directly and just separate between STATIC (for tpp-mlir stack) and SHARED (for plugins to frameworks).